rudder-core/src/test/scala/com/normation/rudder/services/policies/write/WriteSystemTechniquesTest.scala

```diff
@@ -97,23 +107,31 @@ import net.liftweb.common.Loggable
 @RunWith(classOf[JUnitRunner])
 class WriteSystemTechniqueSpec extends Specification with Loggable with ContentMatchers with AfterAll {
 
+  //just a little sugar to stop hurting my eyes with new File(blablab, plop)
+  implicit class PathString(root: String) {
+    def /(child: String) = new File(root + "/" + child)
```

Comment:
Ah, we don't care, tests won't work on windows due to the "/tmp" in all case. And there is a whole lot of hard codede hypothesys that tests, even others, are run on a unix like plateform.